src/translations/en.json

```diff
@@ -1107,15 +1107,15 @@
               },
               "entity_unexpected_unit_energy": {
                 "title": "Unexpected unit of measurement",
-                "description": "The following entities do not have expected units of measurement kWh or Wh:"
+                "description": "The following entities do not have the expected units of measurement ''kWh'' or ''Wh'':"
```

Comment:
We generally use `\"`  to add and escape quotes

Reply:
We seem to use `'` quite a lot, a bit 50/50... I personally like `'` better, but also 🤷‍♂️